.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true,
```

Comment:
when I tried to run eslint, this line triggered an error `"es2021": true,`

Reply:
I'm not sure about the eslint, I did start it in the terminal but seems like not working. I can still use "console.log"